app-samples/Cognitive-Services-Face-WPF/Sample-WPF/Controls/FaceDetectionPage.xaml.cs

```diff
@@ -79,11 +79,21 @@ public partial class FaceDetectionPage : Page, INotifyPropertyChanged
         /// </summary>
         private ObservableCollection<Face> _resultCollection = new ObservableCollection<Face>();
 
+        /// <summary>
+        /// Headpose results container
+        /// </summary>
+        private ObservableCollection<Face> _resultCollectionWithHeadpose = new ObservableCollection<Face>();
+
         /// <summary>
         /// Image used for rendering and detecting
         /// </summary>
         private ImageSource _selectedFile;
 
+        /// <summary>
+        /// Whether draw headpose
```

Comment:
`Whether to draw rectangles rotated by head poses`

Reply:
Done.